### ISSUE TITLE
[1.011.3][Linux] Dead bats can deal damage in multiplayer

### ISSUE THREAD

**Issue by BlackCapCoder** [ekunuke/vagante_issues#154]
Somewhat regularly when a carried dead bat is put down, it will deal the full damage of a live bat. I have also had this happen with a bat being shot dead by an arrow trap and hitting me dead, so the problem might be thrown dead bats.
Also, while carrying a dead bat its sprite occasionally and briefly open its eye. I don't know if this is related.

Neither problem is present in single player.

**Comment by BlackCapCoder** [comment]
It is also possible to take damage from a dead bat held by another player. When this occurred its eye were permanently open and it appeared to be stuck on a frame of the flight animation

**Comment by krompus** [comment]
Confirmed; I've experienced this exact issue, and have taken to avoiding dead eyebats entirely in online multiplayer. It's about 40% of the time.

I'm not sure if the damage taken from the eyebat held by the other player ONLY happens when it shows the "open eye mid-flight" pose, though.

**Comment by BlackCapCoder** [comment]
@krompus Is this on Linux?

**Comment by krompus** [comment]
Yep! I saw the flair (heck, I searched the issues for "Linux" and then found this).
